static/js/theme-map/Renderer/ElementRenderTarget.js

```diff
@@ -47,11 +47,11 @@ class ElementRenderTarget extends RenderTarget {
 
   /**
    * async render(data) => DOMNode
-   * Calls templateFunction(data) and sets value to element.innerHTML, then returns element
+   * Calls templateFunction(data) and sets value to element.textContnet, then returns element
    */
   async render(data) {
     if (this._element) {
-      this._element.innerHTML = this._templateFunction(data);
+      this._element.textContnet = this._templateFunction(data);
```

Comment:
can you manually use this function if it's not used by the test site and verify the behavior is unchanged? I think right now it would fail because of the typo in `textContent`

Reply:
fixed and tested it's working

Reply:
cool! out of curiosity, what did the function do when you called it? did it just run the function you provided and set the return value as the element content?

Reply:
yep, it's used to update the element when data changes